src/index.ts

```diff
@@ -38,12 +36,11 @@ export { default as Fieldset } from './components/atoms/Fieldset/Fieldset';
 export { default as Legend } from './components/atoms/Legend/Legend';
 export { default as Logo } from './components/atoms/Logo/Logo';
 export { default as Trustpilot } from './components/atoms/Trustpilot/Trustpilot';
-
 // Molecules
 export { default as Modal } from './components/molecules/Modal/Modal';
 export { default as ExpiryModal } from './components/molecules/ExpiryModal/ExpiryModal';
 export { default as ZopaFooter } from './components/molecules/ZopaFooter/ZopaFooter';
-export { default as Help, HelpLine } from './components/molecules/Help/Help';
+export { default as Help, HelpLine, details } from './components/molecules/Help/Help';
```

Comment:
This export is the main export of the library. Meaning you will import it like this:
```ts
import { details } from '@zopauk/react-components'
```
I think we should rename this to `HelpLineDetails` or something like that